sonar-batch/src/main/java/org/sonar/batch/cache/ProjectCacheSynchronizer.java

```diff
@@ -72,58 +73,67 @@ public void load(String projectKey, boolean force) {
       } else {
         LOG.info("-- Found project [{}] cache [{}], synchronizing data..", projectKey, lastSync);
       }
-      cacheStatus.delete(projectKey);
+      cacheStatus.delete();
     } else {
       LOG.info("-- Cache for project [{}] not found, synchronizing data..", projectKey);
     }
 
     loadData(projectKey);
-    saveStatus(projectKey);
+    saveStatus();
   }
 
-  private void saveStatus(String projectKey) {
-    cacheStatus.save(projectKey);
+  private void saveStatus() {
+    cacheStatus.save();
     LOG.info("-- Succesfully synchronized project cache");
   }
 
   private void loadData(String projectKey) {
     Profiler profiler = Profiler.create(Loggers.get(ProjectCacheSynchronizer.class));
+    ProjectRepositories projectRepo = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "projectRepo". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1854)